From 67bf190e923d2c76d0755fd07521fb2e02490c0c Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 23 Jun 2017 16:23:41 +0800
Subject: [PATCH 2044/5242] MLK-15149-02 ARM64: dts: Add power domain node in
 dts for i.mx8mq

commit  682425f548c0f9b6e36157343f149e8e8f2d0a54 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add power domain node on i.MX8MQ.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 7eeb5a8..be6d5f5 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -142,6 +142,12 @@
 		};
 	};
 
+	power: power-controller {
+		compatible = "fsl,imx8mq-pm-domain";
+		num-domains = <14>;
+		#power-domain-cells = <1>;
+	};
+
 	gpio1: gpio@30200000 {
 		compatible = "fsl,imx8mq-gpio", "fsl,imx35-gpio";
 		reg = <0x0 0x30200000 0x0 0x10000>;
-- 
1.7.9.5

